settings.gradle.kts

```diff
@@ -18,6 +18,7 @@ pluginManagement {
 include(":plugin",
         ":samples:junit-tests",
         ":samples:kotlin-android",
+        ":samples:kotlin-android-with-moshi-kotlin-codegen",
```

Comment:
I'd rather have a `kotlin-android-moshi-codegen` just to keep it shorter

Reply:
I've just find out that we're ignoring all the hidden files and so adding a gitignore file would have no effect.
I'm going to ignore specific hidden directories (`.idea` and `.gradle`) such that gitignore files can properly be traced by git.